project/Component/nef/Instances/MacFileSytem+Markdown.swift

```diff
@@ -0,0 +1,26 @@
+//  Copyright © 2019 The nef Authors.
+
+import Foundation
+import BowEffects
+import NefMarkdown
+
+extension MacFileSystem: MarkdownSystem {
```

Comment:
I think this conformance should have dependencies reversed. The implementation of `MarkdownSystem` should receive a `FileSystem` as a dependency and use it to save content.